plugin/src/main/resources/data/passives.yml

```
@@ -16,4 +16,13 @@ passives:
-    userFacing: true
\ No newline at end of file
+    userFacing: true
+
+  - id: stampede
+    displayItem: leather
+    userFacing: true
+    metadata:
+      maxLevel: 4
+      ticksPerLevel: 60
+      extraDamagePerLevel: 0.5
+      extraKnockbackPerLevel: 0.1
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)